From c1b2ed4021d22eb6245e25bdd2647ef768a34507 Mon Sep 17 00:00:00 2001
From: Fabricio Silva <hi@fabricio.dev>
Date: Wed, 14 Feb 2024 12:06:58 +0000
Subject: [PATCH] build with dotnet8.0

Build for dotnet8 is not merged to a stable release. For the next release we can probably remove this patch.

---
 src/Directory.Build.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Directory.Build.props b/src/Directory.Build.props
index 8d10c9f9..333301e5 100644
--- a/src/Directory.Build.props
+++ b/src/Directory.Build.props
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <WarningLevel>9999</WarningLevel>
     <ImplicitUsings>enable</ImplicitUsings>
